src/config.json

```diff
@@ -203,64 +174,6 @@
         }
       }
     },
-    {
-      "stock_loan_position" : {
-        "max_objects_per_file" : 50,
-        "file_name" : "stock_loan_positions",
-        "output_file_type": "XML",
-        "output_directory": "out",
-        "upload_to_google_drive": "false",
-        "generation_type": "fixed",
-        "fixed_args": {
-          "record_count" : 50
-        },
-        "dummy_fields": [
-          {
-            "data_type": "string",
-            "data_length": 10,
-            "field_count": 10
-          },
-          {
-            "data_type": "numeric",
-            "data_length": 10,
-            "field_count": 10
-          }
-        ],
-        "file_type_args" : {
-          "xml_root_element" : "stock_loan_positions",
-          "xml_item_name" : "stock_loan_position"
-        }
-      }
-    },
-    {
-      "cash_balance" : {
-        "max_objects_per_file" : 50,
-        "file_name" : "cash_balances",
-        "output_file_type": "XML",
-        "output_directory": "out",
-        "upload_to_google_drive": "false",
-        "generation_type": "fixed",
-        "fixed_args": {
-          "record_count" : 50
-        },
-        "dummy_fields": [
-          {
-            "data_type": "string",
-            "data_length": 10,
-            "field_count": 10
-          },
-          {
-            "data_type": "numeric",
-            "data_length": 10,
-            "field_count": 10
-          }
-        ],
-        "file_type_args" : {
-          "xml_root_element" : "cash_balances",
-          "xml_item_name" : "cash_balance"
-        }
-      }
-    },
```

Comment:
Cash Balance should not be removed (you might have got it mixed up with Cash Flow)